Applications/Spire/Source/KeyBindings/CancelKeyBindingsTableModel.cpp

```diff
@@ -0,0 +1,171 @@
+#include "Spire/KeyBindings/CancelKeyBindingsTableModel.hpp"
+
+using namespace Spire;
+
+namespace {
+  const auto ROW_COUNT = 13;
+  const auto COLUMN_COUNT = 2;
+
+  auto get_action(int index) {
```

Comment:
Is using `auto` as return type for non-public functions preferred as per Spire's coding convention?

Reply:
Yes it is.

Reply:
auto for return types in anonymous namespaces seems to be the convention.